source/hiera/3.0/release_notes.markdown

```diff
@@ -3,6 +3,13 @@ layout: default
 title: "Hiera 3: Release Notes"
 ---
 
+## Hiera 3.0.4
+
+Released October 29, 2015.
+
+Shipped in puppet-agent version 1.2.7.
+
+Hiera 3.0.4 adds support for AIX 5.3.
```

Comment:
Also the only real code change in hiera since 3.0.3 was https://github.com/puppetlabs/hiera/commit/142ef3ff1c327e820bb0e7b7f845df993ded8619, which unfortunately was a maint commit. Might want to mention that the error message is improved when reading hiera data.